static/js/settings_users.js

```diff
@@ -58,7 +58,9 @@ function sort_bot_email(a, b) {
 }
 
 function sort_role(a, b) {
-    return compare_a_b(a.role, b.role);
+    const role_a = people.get_user_type(a.user_id) || "";
+    const role_b = people.get_user_type(b.user_id) || "";
+    return compare_a_b(role_a.toLowerCase(), role_b.toLowerCase());
```

Comment:
We want to sort the role column according to hierarchy of role not alphabetically. So, there is no need to change this.